components/media-object/__tests__/media-object.browser-test.jsx

```diff
@@ -50,7 +50,7 @@ describe(`${MEDIA_OBJECT}: `, () => {
 			expect(container.hasClass('this-is-a-container-test')).to.be.true;
 
 			const body = this.wrapper.find(`.${COMPONENT_CSS_CLASSES.body}`);
-			console.log(body.text());
+			// console.log(body.text());
```

Comment:
Should we get rid of this `console.log`?

Reply:
Yeah, sounds good. I'll update.